dspy/clients/lm.py

```diff
@@ -59,10 +81,65 @@ def __call__(self, prompt=None, messages=None, **kwargs):
         self.history.append(entry)
         
         return outputs
-
+    
     def inspect_history(self, n: int = 1):
         _inspect_history(self, n)
 
+    def launch(self):
+        """Send a request to the provider to launch the model, if needed."""
+        if is_self_hosted_model(self.model):
```

Comment:
let's delete the `self_hosted_model_launch` part, we won't have it ready any time soon.

Reply:
(Deleted, thank you!)